File: sdk/javascript-closure-angular/API/Client/RouteDetailList.js
goog.provide('API.Client.RouteDetailList');

/**
 * @record
 */
API.Client.RouteDetailList = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetailList.prototype.errorcode;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetailList.prototype.errormessage;

/**
 * @type {!number}
 * @export
 */
API.Client.RouteDetailList.prototype.numberofresults;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetailList.prototype.route;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetailList.prototype.timestamp;

/**
 * @type {!Array<!API.Client.RouteDetail>}
 * @export
 */
API.Client.RouteDetailList.prototype.results;

File: sdk/async-scala/src/main/scala/io/swagger/client/model/RouteDetailList.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class RouteDetailList (
  errorcode: Option[String],
errormessage: Option[String],
numberofresults: Option[Integer],
route: Option[String],
timestamp: Option[String],
results: Option[List[RouteDetail]]
)
